runtime/nls/cfre/cfrerr.nls

```diff
@@ -1505,14 +1505,6 @@ J9NLS_CFR_ERR_INVALID_FLAGS_ON_NEW.system_action=The JVM will throw a verificati
 J9NLS_CFR_ERR_INVALID_FLAGS_ON_NEW.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
 
-# <init> is not translatable
```

Comment:
NLS message cannot be deleted as it will change the index number of all the message below.